src/views/Viewer.vue

```diff
@@ -48,6 +48,12 @@
 				@click="showSidebar">
 				{{ t('viewer', 'Open sidebar') }}
 			</ActionButton>
+			<ActionButton
+				v-if="Sidebar && !isSidebarShown"
```

Comment:
It should not be this v-if. You need to check against the permissions :wink:

Reply:
I don't find the field permissions, how can I check?
![grafik](https://user-images.githubusercontent.com/24530680/91013075-73217500-e5e7-11ea-9c01-ef84b2294866.png)

Reply:
That is really weird!

Reply:
@skjnldsv 
Hmm, what can I do?
I want to get this PR done.
Could you push a commit to get the canDelete function done?